managed/ui/src/components/certificates/Certificates.js

```diff
@@ -116,8 +118,61 @@ class Certificates extends Component {
       .finally(() => this.setState({ showSubmitting: false }));
   };
 
+  /**
+   * Delete the root certificate if certificate is safe to remove,
+   * i.e - Certificate is not attached to any universe for current user.
+   * 
+   * @param certificateUUID Unique id of certificate.
+   */
+  deleteRootCertificate = (certificateUUID) => {
+    api.deleteCertificate(certificateUUID).then(
+      () => {
+        this.getCertificateArray()
+      }
+    ).catch(
+      err => {
+        console.log("Error", err)
+      }
+    )
+  };
+
+  /**
+   * Lifecycle method to fetch iniial data required by compoenent
```

Comment:
typos

Reply:
Removed this code, It is no longer needed as we use redux